pkg/securitycontext/provider.go

```diff
@@ -159,6 +159,11 @@ func DetermineEffectiveSecurityContext(pod *api.Pod, container *api.Container) *
 		*effectiveSc.RunAsNonRoot = *containerSc.RunAsNonRoot
 	}
 
+	if containerSc.ReadOnlyRootFilesystem != nil {
```

Comment:
this is a partial commit of https://github.com/kubernetes/kubernetes/pull/22857 for just what I need.  I can rebase https://github.com/kubernetes/kubernetes/pull/22857 after this goes in.